### PR TITLE
feature/VDX-157_sign_vp_callback

### PR DESCRIPTION
**Pull request by hrehman-sphereon**

### CODE REVIEW

**Comment by sksadjad** [comment]
Added usecases for `PresentationSignCallback` @nklomp can you take a look and see if you agree with the way I'm handling it?

**Comment by sksadjad** [comment]
> In this repo it looks okay. Have to check the actual PresentationSign callback in PEX though

it's ok with the latest pex, but it's a local build

File: src/main/PresentationExchange.ts [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
-  EvaluationResults,
-  IPresentationDefinition,
-  KeyEncoding,
-  PEX,
-  PresentationSignCallBackParams,
-  PresentationSignOptions,
-  SelectResults,
-  Status,
-} from '@sphereon/pex';
+import { EvaluationResults, IPresentationDefinition, KeyEncoding, PEX, PresentationSignOptions, SelectResults, Status } from '@sphereon/pex';
```
Comment (nklomp) [inline src/main/PresentationExchange.ts:1]: Github gives these: 

> Argument of type 'PresentationSignCallback' is not assignable to parameter of type '(callBackParams: PresentationSignCallBackParams) => Promise<IVerifiablePresentation>'.

Comment (sksadjad) [reply src/main/PresentationExchange.ts:1]: yeah, as I said it's a local build of pex, I didn't push the local pex library that I'm using

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**